pyhap/accessory_driver.py

```diff
@@ -26,14 +27,15 @@
 import tempfile
 import threading
 import time
-from typing import Optional
+from typing import Optional, List, Dict, Any, Tuple
 
 from zeroconf import ServiceInfo
 from zeroconf.asyncio import AsyncZeroconf
 
 from pyhap import util
 from pyhap.accessory import Accessory, get_topic
-from pyhap.characteristic import CharacteristicError
+from pyhap.characteristic import CharacteristicError, Characteristic
+from pyhap.service import Service
```

Comment:
Needs go below `pyhap.params` import

Or you can do

`isort pyhap/accessory_driver.py`
`black pyhap/accessory_driver.py`